### DISCUSSION TITLE
将alist文件夹路径添加到环境变量以后 直接访问得到的密码和在文件夹中得到的密码不同

### DISCUSSION THREAD

**Discussion by JordanQD** [AlistGo/alist#2923] [Q&A]
![image](https://user-images.githubusercontent.com/25655766/210933441-f232eb68-b2ac-4dcf-a2de-9a8712f7df63.png)
如图

**Comment by anwen-anyi** [comment]
没问题吖，貌似生成的密码是取决于你现在的位置在哪个文件夹他生成到哪里文件夹里面
比如
 - 第一个密码 你在 qiaob 文件夹里面 他会生成一个配置文件密码是 cMHAwLM8
 - 第二个密码 你在 Alist 文件夹 他会生成一个配置文件密码是 QfHtQVcn

建议去和Alist同级目录下运行命令这样 生成的配置文件和Alist运行文件同级 也好找
不然像第一个密码 你这次截图了 下次忘记截图了不知道在哪个文件夹生成的，乱糟糟的 加上突然在某个文件生成一个 /data/ 文件夹有一天你也懵逼是啥....

**Reply by JehovahYF** [reply]
这个问题属实诟病，为啥不能统一初始密码

**Reply by anwen-anyi** [reply]
> 这个问题属实诟病，为啥不能统一初始密码

因为安全问题，以前是统一的，好多人都懒得改 直接一扫端口几十几百个Alist直接去一个一个试试 肯定有能登录的
都玩Alist啦，不至于目录这个问题都搞不清楚吧QAQ.....

**Reply by zzpop** [reply]
“建议去和Alist同级目录下运行命令这样 生成的配置文件和Alist运行文件同级 也好找” 谢谢，我的问题得到解决了！
